### PR TITLE
Create 2024-01-03-Radiant-Capital.md

### PR DESCRIPTION
**Pull request by svg-arch**

### CODE REVIEW

File: content/attacks/posts/2024-01-03-Radiant-Capital.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+ - [0x826d5f4d8084980366f975e10db6c4cf1f9dde6d](https://arbiscan.io/address/0x826d5f4d8084980366f975e10db6c4cf1f9dde6d)
+
+## Losses
+
+The loss amounted to 1902 ETH worth $4.6 million.
+
+## Timeline
+
+- **January 2, 2024, 06:53:38 PM UTC:** The [first malicious](https://arbiscan.io/tx/0x1ce7e9a9e3b6dd3293c9067221ac3260858ce119ecb7ca860eac28b2474c7c9b) transaction occurred.
```
Comment (JediFaust) [inline content/attacks/posts/2024-01-03-Radiant-Capital.md:30]: Fix formatting of time, to match other events

File: content/attacks/posts/2024-01-03-Radiant-Capital.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+- **January 3, 2024, 12:14 AM UTC:** Radiant Protocol [reported](https://twitter.com/RDNTCapital/status/1742338729925112272) the exploit and suspended work on Arbitrum.
+- **January 5, 2024, 02:50 AM UTC:** Radiant Protocol [announced](https://twitter.com/RDNTCapital/status/1743102629411184841) a reward for assistance in investigating the incident.
+- **January 5, 2024, 12:07 AM UTC:** Lending and loan markets on Arbitrum have been [resumed](https://twitter.com/RDNTCapital/status/1743061583692181965).
+- **January 12, 2024:** A [detailed analysis](https://blog.quillaudits.com/trending/radiant-capital-hack-analysis) of the exploit has been published.
+
+## Security Failure Causes
+
+- **Flash Loans Exploitation:** The exploit leveraged flash loans, which allow borrowing large sums, for price manipulation and creating strain on the system.
+- **Smart Contract Vulnerability:** There was a rounding error in the smart contract, which allowed the attacker to artificially increase the profit margin.
```
Comment (JediFaust) [inline content/attacks/posts/2024-01-03-Radiant-Capital.md:39]: Two sections might be combined into one, as Smart Contract Vulnerability

**Review by JediFaust — CHANGES_REQUESTED** [review]

**Comment by svg-arch** [comment]
@JediFaust

**Comment by svg-arch** [comment]
@evgenydmitriev

**Comment by evgenydmitriev** [comment]
@svg-arch, given that we still have requests for changes or co-authored commits for almost all of your pull requests, I suggest you focus on improving the existing pull requests vs opening new ones. It's not a normal practice to use the pull review process for quality assurance purposes. We try to get to all of your PRs weekly, but we have limited resources and we try to spend them on meaningful reviews.

**Review by evgenydmitriev — APPROVED** [review]

**Pull request merged**